Makefile

```
@@ -156,6 +156,39 @@ deps-tidy: ## Run go mod tidy in every module.
+	cd HadesLogManager && go run github.com/swaggo/swag/cmd/swag@$(SWAG_VERSION) init --parseDependency --parseInternal -g main.go -o docs
+
+.PHONY: docs-helm
+docs-helm: ## Regenerate the Helm chart values table (helm/hades/Readme.md) from values.yaml comments.
+	go run github.com/norwoodj/helm-docs/cmd/helm-docs@$(HELM_DOCS_VERSION) \
+		--chart-search-root helm/hades \
+		--template-files Readme.md.gotmpl \
+		--output-file Readme.md \
+		--ignore-non-descriptions
+
+.PHONY: docs-site-sync
+docs-site-sync: ## Sync generated artifacts (OpenAPI specs, helm values) into the Docusaurus site.
```

checkmake (0.3.2)

[warning] Target body for "docs-site-sync" exceeds allowed length of 5 lines (6).

(maxbodylength)